### PR TITLE
Add some minor fixes to code

### PR DESCRIPTION
**Pull request by jorgensd**

### CODE REVIEW

File: src/vasp/automatedPreprocessing/preprocessing_common.py [hunk 1]
```diff
@@ -2,17 +2,19 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 from pathlib import Path
-from typing import Union
+from typing import Union, TYPE_CHECKING
```
Comment (jorgensd) [inline src/vasp/automatedPreprocessing/preprocessing_common.py:5]:
```suggestion
from typing import Union
```

File: src/vasp/automatedPreprocessing/automated_preprocessing.py [hunk 2]
```diff
@@ -212,7 +209,7 @@ def run_pre_processing(input_model, verbose_print, smoothing_method, smoothing_f
         # Extract the region centerline
         refine_region_centerline = []
         info = get_parameters(str(base_path))
-        number_of_regions = info["number_of_regions"]
+        number_of_regions = isave_pathnfo["number_of_regions"]
```
Comment (keiyamamo) [inline src/vasp/automatedPreprocessing/automated_preprocessing.py:212]: Maybe a typo?

**Review by keiyamamo — COMMENTED** [review]
Hi @jorgensd 

Thank you for nice suggestions. It does not pass CI right now due to some minor flake8 issues. But otherwise, it looks great.

File: src/vasp/automatedPreprocessing/automated_preprocessing.py (hunk 2 above)
Comment (jorgensd) [inline src/vasp/automatedPreprocessing/automated_preprocessing.py:212]:
```suggestion
        number_of_regions = info["number_of_regions"]
```

File: src/vasp/automatedPreprocessing/preprocessing_common.py [hunk 3]
```diff
@@ -2,17 +2,19 @@
...
 import meshio
-from vtk import vtkPolyData
 from dolfin import Mesh, MeshFunction, File, HDF5File, FunctionSpace, Function, XDMFFile, cells, Edge
 from vmtk import vmtkdistancetospheres, vmtkdijkstradistancetopoints
 from morphman import vmtkscripts, write_polydata
 
 from vasp.automatedPreprocessing.vmtkmeshgeneratorfsi import vmtkMeshGeneratorFsi
 
+if TYPE_CHECKING:
+    from vtk import vtkPolyData
```
Comment (jorgensd) [inline src/vasp/automatedPreprocessing/preprocessing_common.py:16]:
```suggestion
from vtk import vtkPolyData
```

**Review by keiyamamo — APPROVED** [review]

**Pull request merged**